src/sunburst.js

```diff
@@ -51,7 +50,8 @@ export default Kapsule({
       }
     },
     onClick: { triggerUpdate: false },
-    onHover: { triggerUpdate: false }
+    onHover: { triggerUpdate: false },
+    transitionDuration: { triggerUpdate: false, default: 750 }
```

Comment:
Tiny code style nit, flip the order of the params: `{ default: 750, triggerUpdate: false }`